src/barkhaPJun20/SwappingOfNumbers.java

```diff
@@ -0,0 +1,29 @@
+package barkhaPJun20;
+
+public class SwappingOfNumbers {
+	//Without using third Variable
+	void SwapNumbers(int num1,int num2) {
+		
+		int sum=num1+num2;               //sum=30
```

Comment:
Here we are not supposed to use third variable. U have used sum variable.

Reply:
Hi Swapna, removed third variable as suggested.Please have a look.